### DISCUSSION TITLE
7.6. Replikointiviivettä havaittu tietokantamaster ja -slave palvelinten välillä

### DISCUSSION THREAD

**Discussion by CodoDerDritte** [KohaSuomi/Koha#636] [Huolto- ja häiriötiedotteet]
Tietokantamaster ja -slave palvelinten välillä havaittiin noin klo 14 kasvava viive tietojen replikoinnissa. Viive vaikuttaa siihen etteivät Kohasta otettavat raportit tällä hetkellä ole ajantasaisia. Tietokannasta siivottiin pois muutama sinne selkeästi roikkumaan jäänyt kysely, mutta tilanne ei normalisoitunut sillä. Ongelmasta on oltu yhteydessä konesalioperaattoriin. Tiedotamme lisää tässä keskusteluketjussa kun tiedotettavaa on.

**Comment by CodoDerDritte** [comment]
Käynnistin slave-palvelimen uudelleen, ei vaikutusta.

**Comment by CodoDerDritte** [comment]
Taisipa jäädä odottamaan huomista tämä ongelma, koska operaattori ei nyt ehtinyt hätiin. Huomenna siis jatkuu.

**Comment by lmstrand** [comment]
Viive on alkanut laskea eilen n. 19.30. Poistunut kokonaan n. 22.00.

**Comment by CodoDerDritte** [comment]
Tänään viivettä ei ole enää syntynyt ja sen alkuperäinen syntysyy on mysteeri. Seuraamme edelleen tilannetta, mutta ongelma vaikuttaisi poistuneen.

**Comment by CodoDerDritte** [comment]
Myöskään tänään viive ei enää vaikuta kasvavan, joten asia lienee kunnossa.
